diff --git a/src/index.ts b/src/index.ts
index 790b90c..a72aa18 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -4,6 +4,7 @@ import express, { Express, Request, Response, NextFunction } from 'express';
 import morgan from 'morgan';
 import cron from 'node-cron';
 import config from './config';
+import { localDevelopmentRoutes } from './localDevelopment';
 import Routes from './routes/routes';
 import { syncExchangeRates } from './services/rates';
 import { createServer } from 'http';
@@ -23,7 +24,7 @@ class Relay {
     this.routes = new Routes(this.app);
     this.configure();
     this.invoke();
-    this.startCrons();
+    if (!config.LOCAL_DEV) this.startCrons();
     this.initializeSocketIO();
   }
 
@@ -47,6 +48,7 @@ class Relay {
     this.app.use(morgan('combined'));
     this.app.use(bodyParser.json({ limit: '50mb' }));
     this.app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }));
+    if (config.LOCAL_DEV) this.app.use(localDevelopmentRoutes);
     this.app.all('*', this.checkAuth);
     this.app.use(this.routes.initializeRoutes());
     this.app.use(express.static('public'));
diff --git a/src/services/app.ts b/src/services/app.ts
index fb35c18..cac80c9 100644
--- a/src/services/app.ts
+++ b/src/services/app.ts
@@ -17,6 +17,8 @@ import moment from "moment";
 
 const iap = require("in-app-purchase");
 
+const androidGoogleApi = google.androidpublisher({ version: "v3" });
+if (!config.LOCAL_DEV) {
 google.options({
   auth: new JWT(
     config.GOOGLE_SERVICE_ACCOUNT_EMAIL,
@@ -25,7 +27,7 @@ google.options({
     ["https://www.googleapis.com/auth/androidpublisher"]
   ),
 });
-const androidGoogleApi = google.androidpublisher({ version: "v3" });
+
 
 iap.config({
   appleExcludeOldTransactions: true,
@@ -40,6 +42,8 @@ iap.config({
   verbose: config.ENVIRONMENT === "DEVELOPMENT", // Output debug logs to stdout stream
 });
 
+}
+
 export const createNewApp = async (
   publicId,
   appId,
diff --git a/src/services/messageCentre.ts b/src/services/messageCentre.ts
index 20e9658..fa9a1ce 100644
--- a/src/services/messageCentre.ts
+++ b/src/services/messageCentre.ts
@@ -3,15 +3,16 @@ import { v4 as uuidv4 } from "uuid";
 import db from "../db";
 import { getReleaseTopic, getBroadcastTopic } from "../utils/getReleaseTopic";
 import { firebaseCredential } from "../utils/firebaseCredentials";
+import config from "../config";
 import moment from "moment";
 import { broadcastReleaseNotification } from "./notifications";
 import { NotificationType } from "../interface";
 import * as releaseNotes from "../services/releaseNotes";
 import { Document } from "mongoose";
 
-admin.initializeApp({
-  credential: firebaseCredential(),
-});
+if (!config.LOCAL_DEV) {
+  admin.initializeApp({ credential: firebaseCredential() });
+}
 
 const subscribeToTopic = async (
   topic: string,
